api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.6.x.xml

```diff
@@ -133,5 +133,61 @@
 		<comment>Adding not null constraint to encounter_diagnosis.dx_rank. TRUNK-6081</comment>
 		<addNotNullConstraint tableName="encounter_diagnosis" columnName="dx_rank" columnDataType="int"/>
 	</changeSet>
+
+
+
+	<changeSet id="TRUNK-6206-2023-10-31-1001" author="mujuzi">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="global_property" columnName="view_privilege" />
+			</not>
+		</preConditions>
+		<comment>Adding optional property 'view_privilege' to 'global_property' table</comment>
+		<addColumn tableName="global_property">
+			<column name="view_privilege" type="VARCHAR(255)">
+				<constraints nullable="true" />
+			</column>
+		</addColumn>
+		<addForeignKeyConstraint constraintName="global_property_view_privilege_fk"
+								 baseTableName="global_property" baseColumnNames="view_privilege"
+								 referencedTableName="privilege" referencedColumnNames="privilege" />
+
+	</changeSet>
+
+	<changeSet id="TRUNK-6206-2023-10-31-1002" author="mujuzi">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="global_property" columnName="edit_privilege" />
+			</not>
+		</preConditions>
+		<comment>Adding optional property 'edit_privilege' to 'global_property' table</comment>
+		<addColumn tableName="global_property">
+			<column name="edit_privilege" type="VARCHAR(255)">
+				<constraints nullable="true" />
+			</column>
+		</addColumn>
+		<addForeignKeyConstraint constraintName="global_property_edit_privilege_fk"
+								 baseTableName="global_property" baseColumnNames="edit_privilege"
+								 referencedTableName="privilege" referencedColumnNames="privilege" />
+
+	</changeSet>
+
+	<changeSet id="TRUNK-6206-2023-10-31-1003" author="mujuzi">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="global_property" columnName="delete_privilege" />
+			</not>
+		</preConditions>
+		<comment>Adding optional property 'delete_privilege' to 'global_property' table</comment>
+		<addColumn tableName="global_property">
+			<column name="delete_privilege" type="VARCHAR(255)">
+				<constraints nullable="true" />
+			</column>
+		</addColumn>
+		<addForeignKeyConstraint constraintName="global_property_delete_privilege_fk"
+								 baseTableName="global_property" baseColumnNames="delete_privilege"
+								 referencedTableName="privilege" referencedColumnNames="privilege" />
+
+	</changeSet>
```

Comment:
@dkayiwa  these changes should be in `liquibase-update-to-latest-2.7.x.xml` right?